discovery-provider/src/trending_strategies/ePWJD_trending_tracks_strategy.py

```diff
@@ -47,5 +55,102 @@ def __init__(self):
     def get_track_score(self, time, track):
         return z(time, track)
 
+    def update_track_score_query(self, session):
```

Comment:
I think the possibility for variance in the results is fairly high with this change (unless you've done some spot checking). I think we should roll this out as a "trending experiment," wait for all discovery nodes to get this change, and then dictate the experiment from the client persp. so things stay consistent.

I'd love to avoid doing that if possible, but would want to have a high level of confidence from sandbox nodes that this indeed does produce the same ranking

Reply:
Ahhh gotcha, misinterpreted your other comment my b (also didn't mean to make this a reply, just highlighted the same line :) )

Reply:
Yup, I did spotchecking against prod, but agreed it's much safer to move into another trending experiment for now to verify for a release cycle before full release. 
I don't think it's as bad as you think in terms of adding another 3 columns for karma w/ the other variable - it only takes 11 seconds for the trending_params view to update.